src/main/java/edu/harvard/iq/dataverse/engine/command/impl/DRSSubmitToArchiveCommand.java

```diff
@@ -0,0 +1,383 @@
+
+                if (s3Result == WorkflowStepResult.OK) {
+                    //This will be overwritten if the further steps are successful
+                    statusObject.add(DatasetVersion.ARCHIVAL_STATUS, DatasetVersion.ARCHIVAL_STATUS_FAILURE);
+                    statusObject.add(DatasetVersion.ARCHIVAL_STATUS_MESSAGE, "Bag transferred, DRS ingest call failed");
+
+                    // Now contact DRS
+                    boolean trustCert = drsConfigObject.getBoolean(TRUST_CERT, false);
+                    int jwtTimeout = drsConfigObject.getInt(TIMEOUT, 5);
+                    JsonObjectBuilder job = Json.createObjectBuilder();
+
+                    job.add(S3_BUCKET_NAME, adminMetadata.getString(S3_BUCKET_NAME));
+
+                    job.add(PACKAGE_ID, packageId);
+                    job.add(S3_PATH, spaceName);
+
+                    // We start with the default admin_metadata
+                    JsonObjectBuilder amob = Json.createObjectBuilder(adminMetadata);
+                    // Remove collections and then override any params for the given alias
+                    amob.remove(COLLECTIONS);
+                    // Allow override of bucket name
+                    if (collectionConfig.containsKey(S3_BUCKET_NAME)) {
+                        job.add(S3_BUCKET_NAME, collectionConfig.get(S3_BUCKET_NAME));
+                    }
+
+                    for (Entry<String, JsonValue> entry : collectionConfig.entrySet()) {
+                        if (!entry.getKey().equals(S3_BUCKET_NAME)) {
+                            amob.add(entry.getKey(), entry.getValue());
+                        }
+                    }
+                    job.add(ADMIN_METADATA, amob);
+
+                    String drsConfigString = JsonUtil.prettyPrint(job.build());
+
+                    // TODO - ADD code to ignore self-signed cert
+                    CloseableHttpClient client = null;
+                    if (trustCert) {
+                        // use the TrustSelfSignedStrategy to allow Self Signed Certificates
+                        try {
+                            SSLContext sslContext = SSLContextBuilder.create().loadTrustMaterial(new TrustAllStrategy())
+                                    .build();
+                            client = HttpClients.custom().setSSLContext(sslContext)
+                                    .setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE).build();
+                        } catch (KeyManagementException e) {
+                            // TODO Auto-generated catch block
+                            e.printStackTrace();
+                        } catch (NoSuchAlgorithmException e) {
+                            // TODO Auto-generated catch block
+                            e.printStackTrace();
+                        } catch (KeyStoreException e) {
+                            // TODO Auto-generated catch block
+                            e.printStackTrace();
```

Comment:
AFAIK - it is used widely in the codebase

Reply:
Yeah, I had a brain fart. It's morally equivalent to `System.out.println` which we tend to avoid in place of logger.whatever so we can tell which class it's from. It's probably fine. And yes, fairly widely used.